doc/config.rst

```diff
@@ -303,9 +303,17 @@ Project information
 
    .. versionadded:: 0.5
 
+.. confval:: highlight_options
+
+   A dictionary of options that modify the behavior of the lexer for the
+   selected language. These are lexer-specific. For the options understood by
+   each, see the `Pygments documentation <http://pygments.org/docs/lexers/>`_.
+
```

Comment:
It should be clear that the options only apply to the basic language. I think "for the selected language" is meant to express this, but is not clear to me out of context.